src/installation/guides/arm-devices/index.md

```diff
@@ -0,0 +1,137 @@
+# ARM Devices
+
+Void Linux provides packages and images for several ARM devices. Installing Void
+on such devices can be done in several ways:
+
+- [Pre-built images](#pre-built-images): images that can be flashed directly
+   onto an SD card or other storage medium, but give you a limited partition
+   layout and require manual expansion if you wish to increase the partitions;
+- [Tarball installation](#tarball-installation): PLATFORMFS and ROOTFS tarballs
+   that can be extracted to a previously prepared partition scheme;
+- [Chroot installation](#chroot-installation): follows most of the steps
+   outlined in [the chroot guide](../chroot.md).
+
+Since most of the commands on this guide will be run on external storage, it is
+important to run [sync(1)](https://man.voidlinux.org/sync.1) before removing the
+device.
+
+Further [configuration steps](#configuration) that are mostly specific to such
+devices are also outlined.
+
+## Installation
+
+If you are installing Void Linux on one of the ARM devices covered in the
+"[Supported platforms](./platforms.md)" page, make sure to read its section
+thoroughly.
+
+### Pre-built images
+
+After [downloading and verifying](../../index.md#downloading-installation-media)
+an image, it can be flashed onto an SD card in `/dev/mmcblk0`, for example, with
+the [dd(1)](https://man.voidlinux.org/dd.1) command:
+
+```
+# dd if=<image>.img of=/dev/mmcblk0 bs=4M status=progress
+```
+
+### Preparing your storage
```

Comment:
This section is not on the same logical level as the other 3 (it is necessary information to the tarball/chroot way)

Reply:
Do you have suggestions for where to put it? I agree with you, but I couldn't think of a better placement.

Reply:
Tried a new placement, lmk what you think.

Reply:
suggested section layout:
```
# ARM Devices
<intro>
## Installation
<intro>
### Using pre-built image
<dd command etc>
### Custom partition layout
<intro>
<Preparing your storage>
#### rootfs tarball
#### chroot
## Configuration
...
```

Reply:
This layout is very close to what @flexibeast suggested, so I'm switching to it. Thanks to both :)